Add ARM installer information in GraalVM guide (#7775)
ballerina-platform/ballerina-dev-website: swan-lake/development-tutorials/build-a-graalvm-executable/build-the-executable-locally.md

To complete this part of the guide, you need:
1. Latest [Ballerina Swan Lake](/downloads) distribution

>    > **Note:** If you are using a macOS with ARM64 processor, then install Ballerina using the ARM64 installer available in the [downloads](/downloads) page.

Reviewer: ```suggestion
   > **Note:** If you are using macOS with an ARM64 processor, then, install Ballerina using the [ARM64 installer].
```

Author: ```suggestion
   > **Note:** If you are using macOS with an ARM64 processor, then, install Ballerina using the [ARM64 installer](/downloads).
```
